index.html

```diff
@@ -5,8 +5,33 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="style.css">
+  <script src="https://kit.fontawesome.com/e2597287a1.js" crossorigin="anonymous"></script>
   <title>Portfolio</title>
 </head>
 <body>
+  <div class="app-cont">
+    <div class="header-cont">
+      <div class="nav-cont">
+        <div class="header-img"><img src="images/image_geometry_1.png" alt="">
+        </div>
+        <nav class="nav-bar">
+          <i class="fa-solid fa-bars fa-xl"></i>
+      </div>
+      <header>
+        <h1 class="header-h1">
+          Fuad <br> Nabiyev
+        </h1>
+        <p class="header-p">
+          Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+          If you like what you see and have a project you need coded, don’t hesitate and contact me.
+        </p>
```

Comment:
- [ ]  it would be better if you add font-family and right font-size as in the Figma design in order to. match Figma design
<img width="203" alt="Screen Shot 2022-09-21 at 17 44 33" src="https://user-images.githubusercontent.com/61790587/191563315-3c70d4f1-d656-42f4-98a3-42609e58719b.png">